contrib/ipp/src/pccycbcr44deintw7cn.c

```diff
@@ -83,7 +83,7 @@ extern void  mfxmyNV12ToYUY2_8u_P2C2R(const Ipp8u* pSrcY, int srcYStep,const Ipp
       dst  += 32;
       w += 16;
     }
-    for( w = w; w < width; w += 2 )
+    for( /*w = w*/; w < width; w += 2 )
```

Comment:
don't comment - remove! This assignment don't make any sense.

Reply:
I believe an IPP code author added this for documenting proposes, i.e. to stress that we continue from a current position, so I keep it as a comment.  Sure, it can be deleted.